src/ElementTemplatesIconsRenderer.js

```diff
@@ -0,0 +1,75 @@
+import inherits from 'inherits';
+
+import BaseRenderer from 'diagram-js/lib/draw/BaseRenderer';
+
+import {
+  is
+} from 'bpmn-js/lib/util/ModelUtil';
+
+import {
+  isLabel
+} from 'bpmn-js/lib/util/LabelUtil';
+
+import {
+  append as svgAppend,
+  attr as svgAttr,
+  create as svgCreate
+} from 'tiny-svg';
+
+import {
+  getModelerTemplateIcon
+} from './util';
+
+var HIGH_PRIORITY = 1250;
+
+
+export default function IconsRenderer(
+    bpmnRenderer,
+    config,
```

Comment:
Unused code:

```suggestion
```

Reply:
Correct, I first thought we would need to [keep the configurability](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/draw/BpmnRenderer.js#L868) of the underlying `bpmnRenderer`, but that's not the case.

Reply:
Done.